ee/clickhouse/models/event.py

```diff
@@ -146,10 +147,10 @@ def determine_event_conditions(conditions: Dict[str, Union[str, List[str]]]) ->
             result += "AND timestamp < %(before)s"
             params.update({"before": timestamp})
         elif k == "person_id":
-            result += """AND distinct_id IN (
-                SELECT distinct_id FROM person_distinct_id WHERE person_id = %(person_id)s AND team_id = %(team_id)s
-            )"""
-            params.update({"person_id": v})
+            result += """AND distinct_id IN (%(distinct_ids)s)"""
+            distinct_ids = Person.objects.filter(pk=v)[0].distinct_ids
+            distinct_ids = [distinct_id.__str__() for distinct_id in distinct_ids]
```

Comment:
What was the case where the PG person_id was being passed through? When i try to go to a user page I get this now which looks like a uuid is being passed not a PG person_id
<img width="1178" alt="Screen Shot 2020-10-22 at 11 15 23 AM" src="https://user-images.githubusercontent.com/13127476/96851116-a9cc0e00-1425-11eb-8d48-c8142167104d.png">

Reply:
So, I've changed the person endpoint to explicitely pull from postgres. This will make sure we always ask for an integer ID from the frontend